pkg/kafka/config/producer/config.go

```diff
@@ -12,22 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package config
+package producer
 
 import (
 	"github.com/Shopify/sarama"
 )
 
+// Builder builds a new kafka producer
```

Comment:
Why move this from `26-30` to here? (I feel that this unnecessarily makes the code review bigger)